Geometry/HcalTestBeamData/interface/HcalTB02ParametersFromDD.h

```diff
@@ -18,7 +18,7 @@ class HcalTB02ParametersFromDD {
 
 private:
   static constexpr double k_ScaleFromDDDToG4 = 1.0;
-  static constexpr double k_ScaleFromDD4HepToG4 = 10.0;
+  static constexpr double k_ScaleFromDD4hepToG4 = 10.0;
```

Comment:
@bsunanda Not related to this PR, but this value looks wrong. This file may have been missed in the units conversion. I think it should be:
```
 static constexpr double k_ScaleFromDD4hepToG4 = (1.0 / dd4hep::mm);
```

Reply:
OK Carl I shall take care of this